devices/xiaomi.vacuum/firmwarebuilder/imagebuilder.sh

```diff
@@ -291,8 +291,8 @@ if [ "$UNPROVISIONED" = true ]; then
         chmod +x ./etc/rc.local
         cp $BASEDIR/unprovisioned/wpa_supplicant.conf.wpa2psk ./opt/unprovisioned/wpa_supplicant.conf
 
-        sed -i 's/#SSID#/'$SSID'/g' ./opt/unprovisioned/wpa_supplicant.conf
-        sed -i 's/#PSK#/'$PSK'/g' ./opt/unprovisioned/wpa_supplicant.conf
+        sed -i 's/#SSID#/'"$SSID"'/g' ./opt/unprovisioned/wpa_supplicant.conf
+        sed -i 's/#PSK#/'"$PSK"'/g' ./opt/unprovisioned/wpa_supplicant.conf
```

Comment:
This hunk is wrong and not needed. Did you actually look in the final file?

Reply:
Yes, it was needed otherwise I got an SED error message...

Reply:
You're right, bash is often very confusing :-)